k8s/mongodb/docker/sidecar/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM gcr.io/google-appengine/debian9:latest
```

Comment:
Could you include README to the sidecar directory, similar to this one: https://github.com/GoogleCloudPlatform/mongodb-docker/blob/master/README.md?